packages/mux-audio/README.md

```diff
@@ -47,6 +47,10 @@ Alternatively, use the CDN hosted version of this package:
 <script src="https://unpkg.com/@mux-elements/mux-audio@0.3"></script>
 ```
 
+```html
+<script type="module" src="https://unpkg.com/@mux-elements/mux-audio@0.3/dist/module.mjs"></script>
```

Comment:
`index.mjs` is currently the externalized build that targets bundlers. `module.mjs` has all the depdenencies bundled in so that users won't run into potential issues with `type=module`, like commonjs modules in the chain or requiring the need for import-maps.